### ISSUE TITLE
Default strategy doesn't find .NET core assemblies

### ISSUE THREAD

**Issue by daveriedstra** [jburzynski/TypeGen#2]
Without providing the assemblies property in tgconfig.json, `TypeGen ProjectFolder` outputs the following error:

```
GENERIC ERROR: Value cannot be null.
Parameter name: assemblyFile
```

Adding `"bin/Debug/netcoreapp1.[x]/[ProjectName].dll"` to `assemblies` resolves this. I don't think our project is configured exceptionally. Maybe the default strategy should look there for assemblies?

Project is .NET core 1.1 using VS2015.

**Comment by jburzynski** [comment]
Yes, it's a bug - the search works for .NET Framework, but for .NET Core it searches wrong folders. Will be fixed in the next release.

**Comment by jburzynski** [comment]
Resolved in 1.5.1. Leaving this open in case there are some issues (will close if there are none).
